Button accessibility

CodingItWrong opened the pull request:
This PR makes some buttons in the app more accessible, by marking them as buttons for the sake of accessibility, and by providing text labels for them. The specific use case I'm working with is iOS Voice Control, which lets you tap items on the screen based on their name.

Voice Control provides a names overlay to make it easy to visualize what elements have which names. Before this PR the overlay shows:

![image](https://user-images.githubusercontent.com/15832198/140610284-c0f39d70-8f2b-481a-a32a-3099cc87e16c.png)


Note that:

- Most icon buttons don't have a label
- Days of the month and the year dropdown aren't indicated as buttons at all

After this PR, the overlay shows:

![image](https://user-images.githubusercontent.com/15832198/140610295-6d04c05b-f11e-427c-a9a2-228bc7663450.png)


Note that I haven't added accessibility to all components in the app, just the ones I have experience with from the app I'm using RN Paper in.

Thanks for the great library!

RichardLindhout commented:
This is amazing!! I will accept this if it uses the /src/translations so it'll work across other languages too, could you add this?

CodingItWrong commented:
Will do! I should have time in the morning.

CodingItWrong commented:
@RichardLindhout OK, updated to use the translations API, and added values for US and GB English. As I don't know the other two languages, how should we proceed? TS won't build the app without those values. (I was able to test out the build locally by temporarily deleting those other language files.)

RichardLindhout commented:
Released! Thanks a lot!

RichardLindhout commented:
> @RichardLindhout OK, updated to use the translations API, and added values for US and GB English. As I don't know the other two languages, how should we proceed? TS won't build the app without those values. (I was able to test out the build locally by temporarily deleting those other language files.)

Haha, thanks, I'm doing Google Translate for the Polish version since I don't know the language
